documentation/guides/migrating-from-v5-to-v6.md

```diff
@@ -0,0 +1,251 @@
+
+```diff
+<Modal
+  activator={activator}
+  open={active}
+  onClose={handleChange}
+  primaryAction={{
+    content: 'Add Instagram',
+    onAction: handleChange,
+  }}
+  secondaryActions={[
+    {
+      content: 'Learn more',
+      onAction: handleChange,
+    },
+  ]}
++  title="Reach more shoppers with Instagram product tags"
++  hiddenTitle
+>
+  <Modal.Section>
+    <TextContainer>
+      <p>
+        Use Instagram posts to share your products with millions of
+        people. Let shoppers buy from your store without leaving
+        Instagram.
+      </p>
+    </TextContainer>
+  </Modal.Section>
+</Modal>
+```
+
+### Sheet requires an accessibility label
+
+Accessibility labels are required for screen readers. This new label does not render anything to the page.
+
+```diff
+<Sheet
+  open={sheetActive}
+  onClose={toggleSheetActive}
++ accessibilityLabel="Reach more shoppers with Instagram product tags"
+>
+  <TextContainer>
+    <p>
+      Use Instagram posts to share your products with millions of
+      people. Let shoppers buy from your store without leaving
+      Instagram.
+    </p>
+  </TextContainer>
+</Sheet>
+```
+
+### Replace `ariaPressed` prop with `pressed` in `Button` and `UnstyledButton`
+
+```diff
+- <Button ariaPressed={true}>Let’s go</Button>
++ <Button pressed={true}>Let’s go</Button>
+```
+
+### Removed `Button`'s `stretchContent` prop
+
+Consumers should combine the `fullWidth` and `textAlign="left"` props instead.
+
+```diff
+- <Button stretchContent>Lets go</Button>
++ <Button fullWidth textAlign="left">Lets go</Button>
+```
+
+### Removed `Popover`/`PopoverOverlay`'s `preventAutoFocus` prop
+
+There was duplicated functionality and we have decided to replace `preventAutofocus` with `autofocusTarget="none"`.
+
+```diff
+<Popover
+  active={popoverActive}
+  activator={activator}
+  onClose={togglePopoverActive}
+- preventAutofocus
++ autofocusTarget="none"
+>
+  <p>Hello world</p>
+</Popover>
+```
+
+### Removed the light prop from Tooltip
+
+With the new changes `Tooltip` it now defaults to a light background. The property can safely be removed.
+
+```diff
+<Tooltip
+  content="Hello world"
+- light
+>Order #1001</Tooltip>
+```
+
+### Removed `plain` property in `Pagination`
+
+With the new visual styles we have decided to remove the plain property from `Pagination`. This makes sure the Pagination appears interactive to users.
+
+```diff
+<Pagination
+  hasPrevious
+  hasNext
+  onPrevious={() => handlePrevious()}
+  onNext={() => handleNext()}
+- plain={true}
+/>
+```
+
+### Removed `button-filled-disabled` SASS mixin
+
+This mixin has been removed as `Button` `disabled` property implements this background.
+
+```diff
+.IconButton {
+  padding: 0.8rem;
+- background: button-filled-disabled();
+}
+```
+
+### Removed `plain-button-background` SASS mixin
+
+This mixin has been removed as `Button` `plain` property implements this background.
```

Comment:
```suggestion
This function has been removed as `Button` `plain` property implements this background.
```

Though it feels rubbish to talk "use the Button component" when we're describing the sass api - presumably people using the sass API aren't using our components for some.